circt/circt.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env sh
+
+git clone --depth 1 https://github.com/circt/circt.git
+cd circt
+git submodule update --init --depth 1
+
+cd llvm
```

Comment:
I'd suggest using [the script](https://github.com/llvm/circt/blob/main/utils/build-llvm.sh) in circt purely for consistency's sake. We've validated when building with that script...

Reply:
Sure. Will update in a rebase.